openassessment/xblock/openassessmentblock.py

```diff
@@ -119,9 +119,15 @@ class OpenAssessmentBlock(
     )
 
     allow_file_upload = Boolean(
-        default=False,
+        default=None,
+        scope=Scope.content,
+        help="Do not use. For backwards compatibility only."
+    )
+
+    file_upload_type_raw = String(
```

Comment:
for backward compatibly. if you look at the property definition of file_upload_type, it considers the case that the xblock was created before upgrade.